spec/PhpSpec/Console/ManagerSpec.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace spec\PhpSpec\Console;
+
+use PhpSpec\Console\Manager;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ManagerSpec extends ObjectBehavior
```

Comment:
As would I, and I'll think again to see if there is a way of sorting out the container situation without dealing with the existing IO snafu, but I wasn't able to come up with one originally.  Very happy to try again though, to see if the same advantages can be conferred by some slimmer changeset.